integration/tests/crud/creategetdelete_test.go

```diff
@@ -274,6 +276,110 @@ var _ = Describe("(Integration) Create, Get, Scale & Delete", func() {
 			})
 		})
 
+		Context("can add a nodegroup into a new subnet", func() {
+			var (
+				subnet        *awsec2.Subnet
+				nodegroupName string
+			)
+			BeforeEach(func() {
+				nodegroupName = "test-extra-nodegroup"
+			})
+			AfterEach(func() {
+				cmd := params.EksctlDeleteCmd.WithArgs(
+					"nodegroup",
+					"--verbose", "4",
+					"--cluster", params.ClusterName,
+					nodegroupName,
+				)
+				Expect(cmd).To(RunSuccessfully())
+
+			})
+			It("creates a new nodegroup", func() {
+				cfg := &api.ClusterConfig{
+					Metadata: &api.ClusterMeta{
+						Name:   params.ClusterName,
+						Region: params.Region,
+					},
+				}
+				ctl, err := eks.New(&api.ProviderConfig{Region: params.Region}, cfg)
+				Expect(err).NotTo(HaveOccurred())
+				cl, err := ctl.GetCluster(params.ClusterName)
+				Expect(err).NotTo(HaveOccurred())
+				awsSession := NewSession(params.Region)
+				ec2 := awsec2.New(awsSession)
+				existingSubnets, err := ec2.DescribeSubnets(&awsec2.DescribeSubnetsInput{
+					SubnetIds: cl.ResourcesVpcConfig.SubnetIds,
+				})
+				Expect(err).NotTo(HaveOccurred())
+				Expect(len(existingSubnets.Subnets) > 0).To(BeTrue())
+				s := existingSubnets.Subnets[0]
+
+				cidr := *s.CidrBlock
+				var (
+					i1, i2, i3, i4, ic int
+				)
+				fmt.Sscanf(cidr, "%d.%d.%d.%d/%d", &i1, &i2, &i3, &i4, &ic)
+				cidr = fmt.Sprintf("%d.%d.%s.%d/%d", i1, i2, "255", i4, ic)
+
+				var tags []*awsec2.Tag
+
+				// filter aws: tags
+				for _, t := range s.Tags {
+					if !strings.HasPrefix(*t.Key, "aws:") {
+						tags = append(tags, t)
+					}
+				}
+				output, err := ec2.CreateSubnet(&awsec2.CreateSubnetInput{
+					AvailabilityZone: aws.String("us-west-2a"),
+					CidrBlock:        aws.String(cidr),
+					TagSpecifications: []*awsec2.TagSpecification{
+						{
+							ResourceType: aws.String(awsec2.ResourceTypeSubnet),
+							Tags:         tags,
+						},
+					},
+					VpcId: s.VpcId,
+				})
```

Comment:
does this need its own unique cleanup for the `DeleteCluster` to work? or is it alright?

Reply:
That's a good question. I didn't try that out. I'll try it out.

Reply:
worked! :) 

Added the subnet:

```
2021-08-02 15:21:06 [ℹ]  node "ip-192-169-16-174.us-west-2.compute.internal" is ready
2021-08-02 15:21:06 [ℹ]  node "ip-192-169-254-82.us-west-2.compute.internal" is ready
2021-08-02 15:21:06 [✔]  created 1 managed nodegroup(s) in cluster "test-adding-subnets-9"
2021-08-02 15:21:08 [ℹ]  checking security group configuration for all nodegroups
2021-08-02 15:21:08 [ℹ]  all nodegroups have up-to-date configuration
```

Deleted the cluster:

```
2021-08-02 15:30:33 [ℹ]  waiting for CloudFormation stack "eksctl-test-adding-subnets-9-nodegroup-managed-ng-1"
2021-08-02 15:30:51 [ℹ]  waiting for CloudFormation stack "eksctl-test-adding-subnets-9-nodegroup-managed-ng-1"
2021-08-02 15:31:09 [ℹ]  waiting for CloudFormation stack "eksctl-test-adding-subnets-9-nodegroup-managed-ng-1"
2021-08-02 15:31:10 [ℹ]  will delete stack "eksctl-test-adding-subnets-9-cluster"
2021-08-02 15:31:11 [✔]  all cluster resources were deleted

```